doc/api/buffer.md

```diff
@@ -618,6 +618,10 @@ Returns the actual byte length of a string. This is not the same as
 [`String.prototype.length`] since that returns the number of *characters* in
 a string.
 
+*Note* that for `'base64'` and `'hex'`, this function assumes valid input. For
+strings that do not contain Base64/Hex-encoded data, the return value might be
```

Comment:
nit: Maybe mention that the method doesn’t account for whitespace in the input … for some people that might be included in “invalid”, but that’s kind of subjective…